mdd/python/translation/openconfig_xe/xe_qos.py

```diff
@@ -0,0 +1,280 @@
+# -*- mode: python; python-indent: 4 -*-
+from translation.openconfig_xe.common import xe_system_get_interface_ip_address
```

Comment:
xe_system_get_interface_ip_address isn't used in the file

Reply:
Done